src/legacy/core_plugins/kbn_doc_views/public/views/table.html

```diff
@@ -81,6 +81,11 @@
           class="kbnDocViewer__field">
       </td>
       <td>
+        <span ng-if="enableDocValueCollapse(field)" aria-expanded="{{!!fieldRowOpen[field]}}">
```

Comment:
QQ: Can these attributes not be applied directly to the `button` instead of creating this extra wrapper?

Reply:
yes, I will make a fix